src/RestServer/RestServer.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Neo.Plugins.RestServer</PackageId>
+    <RootNamespace>Neo.Plugins.RestServer</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>disable</Nullable>
```

Comment:
```suggestion
    <Nullable>enable</Nullable>
```

Reply:
We will need to fix all the warning too or errors.